bme680_driver:
  ros__parameters:

    # The I2C bus number.
    i2c_bus: 1

    # The I2C address of the BME680 sensor.
    i2c_address: 0x77

    # The update rate of the sensor in Hz.
    update_rate: 1
